GitUI/Editor/BlameAuthorMargin.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Windows.Forms;
+using GitExtUtils.GitUI;
+using ICSharpCode.TextEditor;
+using Microsoft.VisualStudio.Threading;
+
+namespace GitUI.Editor
+{
+    public class GitBlameDisplay
+    {
+        public JoinableTask<Image> Avatar { get; set; }
+        public int RecencyIndex { get; set; }
```

Comment:
I don't believe there is an English word "recency".
May be call it "AgeBucket" - "AgeBucketIndex", "AgeBucketColor", etc.?